README.md

```diff
@@ -57,11 +57,11 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
```

Comment:
- [ ] you used the new template readme file but it's empty please make sure to customize it for this  project without letting the sections empty